src/test/regress/expected/multi_create_table.out

```diff
@@ -160,39 +160,206 @@ SELECT repmodel FROM pg_dist_partition WHERE logicalrelid='mx_table_test'::regcl
 (1 row)
 
 DROP TABLE mx_table_test;
-RESET citus.replication_model;
--- Show that it is not possible to create an mx table with the old 
--- master_create_distributed_table function
-CREATE TABLE mx_table_test (col1 int, col2 text);
-SELECT master_create_distributed_table('mx_table_test', 'col1', 'hash');
+-- Show that master_create_distributed_table ignores citus.replication_model GUC
+CREATE TABLE s_table(a int);
+SELECT master_create_distributed_table('s_table', 'a', 'hash');
+NOTICE:  using statement based replication
+DETAIL:  master_create_distributed_table is deprecated and does not support the current replication_model setting
+HINT:  Use create_distributed_table to create streaming replicated tables.
  master_create_distributed_table 
 ---------------------------------
  
 (1 row)
 
-SELECT repmodel FROM pg_dist_partition WHERE logicalrelid='mx_table_test'::regclass;
+SELECT repmodel FROM pg_dist_partition WHERE logicalrelid='s_table'::regclass;
  repmodel 
 ----------
  c
 (1 row)
 
-DROP TABLE mx_table_test;
--- Show that when replication factor > 1 the table is created as coordinator-replicated
+-- Show that master_create_worker_shards complains when RF>1 and replication model is streaming
+UPDATE pg_dist_partition SET repmodel = 's' WHERE logicalrelid='s_table'::regclass;
+SELECT master_create_worker_shards('s_table', 4, 2);
+ERROR:  using replication factor 2 with streaming replicated tables are not supported
+DETAIL:  When master_create_distributed_table is called with citus.replication_model streaming, then the table is marked for streaming replication and the shard replication factor of streaming replicated tables must be 1.
```

Comment:
> When master_create_distributed_table is called with citus.replication_model streaming, then the table is marked for streaming replication 

This seems inaccurate.

Apart from catalog updates, could a user ever hit this message?